behave/features/steps/cla_in_scope.py

```diff
@@ -0,0 +1,50 @@
+from behave import *
+from selenium.webdriver.support.ui import WebDriverWait
+from features.constants import CLA_PUBLIC_URL
+
+def assert_header_on_page(title, context):
+    heading = context.helperfunc.find_by_xpath('//h1')
+    assert heading is not None
+    assert heading.text == title
+    
+def wait_until_page_is_loaded(path, context):
+    def do_test(*args):
+        return context.helperfunc.get_current_path() == path
+    wait = WebDriverWait(context.helperfunc.driver(), 10)
+    wait.until(do_test)
+     
+@given(u'I have selected the start now button on the start page')
+def step_start_page(context):
+    start_page_url = f"{CLA_PUBLIC_URL}"
+    context.helperfunc.open(start_page_url)
+    assert_header_on_page("Check if you can get legal aid", context)
+    start_button = context.helperfunc.find_by_id('start')
+    assert start_button is not None
+    assert start_button.text == "Start now"
+    start_button.click()
+
+@given(u'I am on the scope diagnosis page')
+def step_scope_page(context):
+    wait_until_page_is_loaded("/scope/diagnosis/", context)
+    assert context.helperfunc.get_current_path() == "/scope/diagnosis/"
```

Comment:
Do we need this additional assert of the path as it is done by `wait_until_page_is_loaded` call above.

Reply:
Resolved by commit 8fc4c2